### PR TITLE
CLDR 44.1 Update - Units

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [ ] `ReleaseNotes` has added or is not needed.
* [ ] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [ ] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Update unitfmt.json files cldr v44.1
* Fix test failres and add one more test case
* Fix tool code to filter out more cases (10p-27, 10p-30, 10p27, 10p30)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/data/locale/en/GB/unitfmt.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
             "dessert-spoon": "one#{n} US dessertspoon|#{n} US dessertspoons",
             "dessert-spoon-imperial": "one#{n} dessertspoon|#{n} dessertspoons",
             "dot": "one#{n} dot|#{n} dots",
-            "dot-per-centimeter": "one#{n} dot per centimetre|#{n} dots per centimetre",
+            "dot-per-centimeter": "one#{n} dot per centimeter|#{n} dots per centimeter",
```
Comment (ehoogerbeets) [inline js/data/locale/en/GB/unitfmt.json:47]: This is definitely not right. British spelling is "centimetre" like it was before. Are you sure this is what it says in CLDR?

**Review by ehoogerbeets — COMMENTED** [review]
Please check if this is what CLDR says. (If so, then CLDR is wrong!)

File: js/data/locale/en/GB/unitfmt.json (hunk 1 above)
Comment (gouniLee) [reply js/data/locale/en/GB/unitfmt.json:47]: Yes It's changes from cldr v44.1 (cldr-units-full/main/en-GB/units.json)
In units.json of CLDR, it has `area-square-centimeter`, `graphics-pixel-per-centimeter`, `graphics-dot-per-centimeter`, `length-centimeter`, and `volume-cubic-centimeter`. 
others still have `centimetre` but only `graphics-dot-percentimeter` value has changed t `centimeter` in this version.

Comment (ehoogerbeets) [reply js/data/locale/en/GB/unitfmt.json:47]: So stupid... why don't they have integrity checks? Spelling it one way in one unit and a different way in another unit is not a preferential thing. Anyways, we will have to wait until a future version of CLDR to get it corrected, I supposed.

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**